docker-compose.prod.yml

```diff
@@ -21,10 +21,20 @@ services:
     depends_on:
       - server
       - webapp
+    deploy:
+      restart_policy:
+        condition: on-failure
+        delay: 5s
```

Comment:
In terms of `delay` and `window` I think the default zero value is better let the container restart immediately.